.template-helpers/variant.md

```diff
@@ -0,0 +1,7 @@
+# Image Variants
+
+The `%%REPO%%` images come in many flavors, each designed for a specific use case.
```

Comment:
I would argue that this is *one* image with many flavors, hence: "The `%%REPO%%` image come in many flavors".